sampleproject/sampleproject/settings.py

```diff
@@ -80,6 +80,14 @@
 
 WSGI_APPLICATION = "sampleproject.wsgi.application"
 
+# COMPONENTS = {
```

Comment:
Added example of the django-components settings into the sampleproject to help people get started